GVRf/Framework/framework/src/main/jni/objects/shader_data.cpp

```diff
@@ -68,7 +68,7 @@ void ShaderData::setTexture(const char* key, Texture* texture)
         if (temp.compare(key) == 0)
         {
             Texture* oldtex = mTextures[i];
-            makeDirty(oldtex ? MOD_TEXTURE : NEW_TEXTURE);
+            makeDirty(((oldtex != nullptr) && (texture != nullptr)) ? MOD_TEXTURE : NEW_TEXTURE);
```

Comment:
What is the difference between MOD_TEXTURE and NEW_TEXTURE? Also I don't see obvious use of MOD_TEXTURE.